x-pack/plugins/security_solution/server/lib/index_fields/types.ts

```diff
@@ -16,4 +17,6 @@ export interface IndexFieldDescriptor {
   type: string;
   searchable: boolean;
   aggregatable: boolean;
+  esTypes?: string[];
```

Comment:
Why is `esTypes` optional. I think `getFieldsForWildcard()` always returns it. https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/server/lib/index_fields/elasticsearch_adapter.ts#L42